examples/delete_database.py

```diff
@@ -27,7 +27,7 @@ def run(database: str):
 
 if __name__ == "__main__":
     p = ArgumentParser()
-    p.add_argument("id", type=str, nargs=1, help="database id")
+    p.add_argument("id", type=str, nargs=1, help="database name")
```

Comment:
not a big deal, but should we change the argument name to "name" to match the help text?

Reply:
oops right